python/tchannel/tornado/message_factory.py

```diff
@@ -56,6 +58,30 @@ def __init__(self, remote_host=None, remote_host_port=None):
         self.in_checksum = {}
         self.out_checksum = {}
 
+    def build_raw_error_message(self, err):
```

Comment:
What is `err`? An `ErrorMessage`? A `ProtocolError `? We have a lot of name collisions happening.